.eslintrc.js

```diff
@@ -92,11 +89,24 @@ module.exports = {
       }
     },
     {
-      files: ['**/govuk/components/**/*.{cjs,js,mjs}'],
-      excludedFiles: ['**/govuk/components/**/*.test.{cjs,js,mjs}'],
+      files: ['src/govuk/**/*.{cjs,js,mjs}'],
```

Comment:
I expanded the downgrade of EcmaVersion to all files in `src/govuk`, meaning this now includes `all.mjs`, `common.mjs` and `i18n.mjs`. This should ensure all code ending up in the bundle (besides the polyfills) gets parsed as ES2015 by ESLint.